docs/how-to-deploy-bold-bi-using-existing-db-server.md

```diff
@@ -7,18 +7,11 @@ In the following section, we are going to run the Bold BI application by passing
    ```sh
    curl -o docker-compose.yml "https://raw.githubusercontent.com/boldbi/boldbi-docker/main/deploy/single-container-with-env-variable/docker-compose.yml"
    ```
-3. Open the docker compose file and fill the mandatory fields - <b>APP_BASE_URL</b>, <b>Unlock Key</b> and <b>Database details</b>
+3. Open the docker compose file and fill the mandatory fields - <b>Unlock Key</b> and <b>Database details</b>
```

Comment:
You should use different compose file for each use case? Do we have a separate compose file with above mentioned screenshot?

Reply:
Yes, we have separate for this deployment file.
Docker-compose yaml file => https://raw.githubusercontent.com/SivanesanSyncfusion/boldbi-docker-old/main/deploy/single-container-with-env-variable/docker-compose.yml

And also we have used separate yaml file for different use case.